tests/test_dspace_dim.py

```diff
@@ -0,0 +1,294 @@
+                description=(
+                    "A low resolution version of this movie was published as supplemental"
+                    " material to: Rieder, C. L. and A. Khodjakov. Mitosis through the "
+                    "microscope: advances in seeing inside live dividing cells. Science "
+                    "300 (2003): 91-96, doi: 10.1126/science.1082177"
+                ),
+                item_type=None,
+                relationship=None,
+                uri=None,
+            ),
+            RelatedItem(
+                description="International Association of Aquatic and Marine Science "
+                "Libraries and Information Centers (38th : 2012: Anchorage, Alaska)",
+                item_type=None,
+                relationship="ispartofseries",
+                uri=None,
+            ),
+            RelatedItem(
+                description=None,
+                item_type=None,
+                relationship=None,
+                uri="https://doi.org/10.1002/2016JB013228",
+            ),
+        ],
+        rights=[
+            Rights(
+                description="Attribution-NonCommercial-NoDerivatives 4.0 International",
+                kind=None,
+                uri=None,
+            ),
+            Rights(
+                description=None,
+                kind=None,
+                uri="http://creativecommons.org/licenses/by-nc-nd/4.0/",
+            ),
+            Rights(description="CC-BY-NC 4.0", kind="license", uri=None),
+        ],
+        subjects=[
+            Subject(
+                value=["Spermatocyte", "Microtubules", "Kinetochore microtubules"],
+                kind="lcsh",
+            ),
+            Subject(
+                value=["Polarized light microscopy", "LC-PolScope"],
+                kind="Subject scheme not provided",
+            ),
+        ],
+        summary=[
+            (
+                "The events of meiosis I in a living spermatocyte obtained from the "
+                "testis of a crane-fly larva are recorded in this time-lapse sequence "
+                "beginning at diakinesis through telophase to the near completion of "
+                "cytokinesis following meiosis I."
+            )
+        ],
+    )
+
+
+def test_dspace_dim_record_optional_fields_blank_transforms_correctly(
+    dspace_dim_record_partial, dspace_dim_record_optional_fields_blank
+):
+    output_records = dspace_dim_record_partial(
+        input_records=dspace_dim_record_optional_fields_blank
+    )
+    assert next(output_records) == TimdexRecord(
+        source="A Cool Repository",
+        source_link="https://example.com/oai:darchive.mblwhoilibrary.org:1912/2641",
+        timdex_record_id="cool-repo:oai:darchive.mblwhoilibrary.org:1912-2641",
+        title=(
+            "Time lapse movie of meiosis I in a living spermatocyte from the crane "
+            "fly, Nephrotoma suturalis, viewed with polarized light microscopy"
+        ),
+        alternate_titles=None,
```

Comment:
Not a big deal, but we don't really need to put all these `None` arguments in since that's the default for all non-required fields.